.github/actions/deploy-klt-on-cluster/action.yml

```
@@ -99,7 +99,19 @@ runs:
         echo "Installing Keptn using helm"
         helm version
 
-        cd ./chart
+        cd ./lifecycle-operator/chart 
+        helm dependency update
+        helm dependency build 
+        
+        cd ../../metrics-operator/chart 
+        helm dependency update
+        helm dependency build 
+        
+        cd ../../klt-cert-manager/chart 
+        helm dependency update
+        helm dependency build
+        
+        cd ../../chart
         helm dependency update
         helm dependency build
```

Let's use a for loop here, something like the following

```suggestion
        for dir in lifecycle-operator metrics-operator klt-cert-manager chart
do
    cd ./$dir/chart
    helm dependency update
    helm dependency build
    cd -
done
```

---

I am not sure I want to keep the dependency build here after releasing the common chart... it is not like we will add much more to it 🤔